File: 771.JewelsandStones/Solution.go
//ref
//https://stackoverflow.com/questions/18130859/how-can-i-iterate-over-a-string-by-runes-in-go
func numJewelsInStones(J string, S string) int {
	count := 0
	for _, j := range J {
		for _, s := range S {
			if s == j {
				count++
			}
		}
	}
	return count
}

File: 771.JewelsandStones/Solution.java
//ref
//https://stackoverflow.com/questions/3799130/how-to-iterate-through-a-string
class Solution {
    public int numJewelsInStones(String J, String S) {
        int count = 0;
        for(char j : J.toCharArray()){
            for(char s : S.toCharArray()){
                if(j==s){
                    count++;
                }
            }
        }
        return count;
    }
}